### PR TITLE
Custom fields validation double errors (#709)

### PR DESCRIPTION
**Pull request by linki**
fixes the display of multiple error messages of the same attribute for the same custom field (fixes #709)

### CODE REVIEW

**Comment by mfrister** [comment]
According to Travis a few tests possibly related to validation fail. Please fix them :)

**Comment by mfrister** [comment]
Seems Project.issue_custom_fields was also renamed :(
And a tests:unit test is failing.

**Comment by linki** [comment]
I will look into it tomorrow

File: test/unit/project_test.rb [hunk 1]
```diff
@@ -263,7 +263,7 @@ def test_set_parent_should_add_children_in_alphabetical_order
   end
 
   def test_rebuild_should_sort_children_alphabetically
-    ProjectCustomField.delete_all
+    CustomField.destroy_all
```
Comment (linki) [inline test/unit/project_test.rb:266]: anyone has a problem with that?

**Comment by mfrister** [comment]
According to #459, this PR is no longer necessary.

**Pull request closed without merging**